mlflow/types/schema.py

```diff
@@ -442,6 +442,7 @@ class Array:
     def __init__(
         self,
         dtype: Union["Array", "Map", DataType, Object, str],
+        is_sparkml_vector: bool = False,
```

Comment:
What about subclassing Array class instead? For many use cases out of spark, this field and accessors are useless and may be confusing. Also it reduces the risk of regressing those use cases drastically.

Reply:
This is a good idea. Let me update it

Reply:
@B-Step62 Updated according to your suggestion